front/app/components/FormBuilder/edit/utils.ts

```diff
@@ -17,6 +17,36 @@ const reorder = <ListType>(
   return result;
 };
 
+// Mapping of community monitor page keys to their category keys
+const categoryPageKeyMapping: Record<string, string> = {
+  page_quality_of_life: 'quality_of_life',
+  page_service_delivery: 'service_delivery',
+  page_governance_and_trust: 'governance_and_trust',
+};
```

Comment:
Would it be nicer if we added the category to the pages in the default fields and then there is at least a more direct link. Probably not a big change